index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porfolio project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <div class="header-section">
+<a href="#" class="logo">My Logo</a>
+<nav> <img src="images/Union.png" alt=""></nav>
+<!-- <a class="porfolio"href="#">Porfolio</a>
+<a href="#about" class="about">About</a>
+<a href="#contact" class="contact">Contact</a> -->
+</div>
+<section class="main-section">
+    <h1>I'm Therese TUYISABE<br> Glad to see you</h1>
+    <div class="primary text">
+<p class="text1">
+I am software developer!I can help you build a product feature or 
+website lookthrough sameof my work and experience!
+if you like what you see and have a project you need coded don't hestiate to contact me.
+</p>
+<p class="text2">Let's Connect</p></div>
```

Comment:
This should be a `<button>` or `<a>' . Apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href.